### ISSUE TITLE
Webhook ENABLE option works the oposite

### ISSUE THREAD

**Issue by daniel-lucio** [Drakkar-Software/OctoBot-Tentacles#615]
My user/config.json file has the following
        "webhook": {
            "enabled": true,
            "ngrok-token": "Empty",
            "webhook-bind-ip": "192.168.7.106",
            "webhook-bind-port": 7777
        }

2021-11-20 01:24:23 INFO   ServiceFactory       service_factory.py:54       WebHookService can't be initialized: configuration is missing, wrong or incomplete. This is normal if you did not configure it yet.

I have tried several variants of the json:
  "ngrok-token": "",

non-existen  "ngrok-token line
"webhook-bind-ip": 0.0.0.0
"webhook-bind-ip": 127.0.0.1

------------------
However, putting
        "webhook": {
            "enabled": false
            "ngrok-token": "Empty",
            "webhook-bind-ip": "192.168.7.106",
            "webhook-bind-port": 7777
        }

Returns:
 2021-11-20 01:50:53,671 INFO     WebHookService       Webhook configured on address: 192.168.7.106 and port: 7777

Anyway, port 7777 is still closed.

My wild guess is the error is here https://github.com/Drakkar-Software/OctoBot-Tentacles/blame/master/Services/Services_bases/webhook_service/webhook.py around line 86. 

Not a python coder :(

**Comment by valouvaliavlo** [comment]
"Enabled" in JSON is to enable ngrok. The last config seem to be ok

**Comment by daniel-lucio** [comment]
> "Enabled" in JSON is to enable ngrok. The last config seems to be ok

Yes, that's what I was reading the code (maybe a better wording).

However, when Enabled is false, the port is closed.

**Comment by valouvaliavlo** [comment]
Have your enable "TradingViewSignalsTradingMode" ?

**Comment by valouvaliavlo** [comment]
> Oh, it is open now, This has to be added to the documentation.
> 
> The idea is not to use tradeview and let one code it sown signalling code.

It's not dependant of TradingViewTradingMode, you can code your own. It just need a subscriber

**Comment by valouvaliavlo** [comment]
https://github.com/Drakkar-Software/OctoBot-Tentacles/blob/70b56b6ab732ff1f37c40522d467d100b84c1acf/Trading/Mode/trading_view_signals_trading_mode/trading_view_signals_trading.py#L60

**Comment by valouvaliavlo** [comment]
https://github.com/Drakkar-Software/OctoBot-Tentacles/blob/70b56b6ab732ff1f37c40522d467d100b84c1acf/Trading/Mode/trading_view_signals_trading_mode/trading_view_signals_trading.py#L60-L87
